credentialdigger/download.py

```diff
@@ -63,8 +66,8 @@ def download(model, *pip_args):
         # Dirty, but since spacy.download and the auto-linking is
         # mostly a convenience wrapper, it's best to show a success
         # message and loading instructions, even if linking fails.
-        print(
-            "Download successful but linking failed",
+        logger.error(
+            "%s\n%s","Download successful but linking failed",
             "Creating a shortcut link for %s didn't work (maybe you "
             "don't have admin permissions?)" % model
         )
```

Comment:
"%s\n%s" there is not correct, it will not print anything. 
Leave the same message there was before (and adjust it with f-strings please)